Raven.Database/Raft/ClusterManagementHttpClient.cs

```diff
@@ -210,6 +211,34 @@ public Task SendDatabaseDeleteAsync(string databaseName, bool hardDelete)
             }
         }
 
+        public Task SendReplicationStateAsync(Dictionary<string, Tuple<DateTime,string>> databaseToLastModified)
```

Comment:
Let us avoid `Tuple<DateTime,string>`. No idea what the string key on the dic is (assume db name), and certainly no clue what the `Item2` in the tuple is meant to be.